TestFoundation/TestNSCharacterSet.swift

```diff
@@ -17,7 +15,93 @@ import SwiftFoundation
 import SwiftXCTest
 #endif
 
+private struct Box {
```

Comment:
Is this `Box` necessary? This doesn't seem to be significantly less work than just replicating some tests:

```swift
for (lhs, rhs) in pairs {
    XCTAssertEqual(NSCharacterSet(charactersIn: lhs), NSCharacterSet(charactersIn: rhs))
    XCTAssertEqual(CharacterSet(charactersIn: lhs), CharacterSet(charactersIn: rhs))
}
```